react/components/Title/index.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+import { LEFT, RIGHT, CENTER } from './constants.js'
+import title from './title.css'
+
+/**
+ * Name component. Show name and relevant SKU information of the Product Summary
+ */
+class Title extends Component {
+  static propTypes = {
+    /** Text to be displayed */
+    content: PropTypes.string,
+    /** Alignment of the title */
+    alignment: PropTypes.oneOf([LEFT, RIGHT, CENTER]).isRequired,
+  }
+
+  static defaultProps = {
+    alignment: CENTER,
+  }
+
+  render() {
+    const { content, alignment } = this.props
+
+    return (
+      <div className={`${title.titleContainer} flex justify-between`}>
+        <div className={`${title.titleBox} w-100 ma1 c-muted-1`}>
+          {alignment == LEFT && <h3 className="tl t-heading-3">{content}</h3>}
```

Comment:
Okay... I think this would be cleaner if you just:
```js
  ...
   import classNames from 'classnames'
  ...
  cosnt titleClasses = classNames(styles.title, 't-heading-3', {
    tl: align === LEFT,
    tc: align === CENTER,
    tr: align === RIGHT
  })
  ...
  <h3 className={titleClasses}>{content}</h3>
  ...
```

Reply:
Thank you for this! Done. :smile: